frontendActions/build.gradle

```diff
@@ -10,6 +10,8 @@ dependencies {
   implementation project(':frontendDataKeys')
 }
 
+applySubtypingChecker()
```

Comment:
It's not really a problem if someone forgets about this line in build.gradle of a module that uses some `is...`/`as...` methods... coz then Checker is gonna spit out some weiiiird errors that I don't want to understand